projects/NodeBB/services/nodebb/examples/postgresql.nix

```diff
@@ -0,0 +1,32 @@
+{ pkgs, ... }:
+
+{
+  services.nodebb = {
+    enable = true;
+    admin = {
+      username = "admin";
+      email = "admin@example.com";
+      passwordFile = pkgs.writeText "nodebb-admin-password" "nodebb";
```

Comment:
Let's add a big fat WARNING here that this is merely for demonstration and people should use proper secret management.

Reply:
The example must be self-contained, how else would we guarantee it to work on the consumer end? If we leave it open there's a huge gap in the instructions to get it to run